app/src/main/java/com/example/paetz/yacguide/network/RockParser.java

```diff
@@ -109,7 +111,9 @@ private void parseRoutes(String json) throws JSONException {
         for (final Rock rock : db.rockDao().getAll(_sectorId)) {
             boolean ascended = false;
             for (final Route route : db.routeDao().getAll(rock.getId())) {
-                ascended |= (route.getAscendCount() > 0);
+                for (final Ascend ascend : db.ascendDao().getAscendsForRoute(route.getId())) {
+                    ascended |= (ascend.getStyleId() < AscendTypes.BAILED);
```

Comment:
Info about ascend type could be stored in Route db field ascendCount as binary digits to prevent iteration over (probably lots of) ascends, e.g. 0b1001 = ascended (1), not sacked (0), not watched (0), projected (1). Same for RouteActivity. --> Done in follow-up ticket #52.